backend/app/models.py

```diff
@@ -1,3 +1,58 @@
 """
 Models for the Self-Sustaining Cities web app.
 """
+
+from django.db import models
+
+
+class Location(models.Model):
+    """
+    Location model
+    Fields:
+        `name`: name of the location
+    """
+    name = models.CharField(max_length=128)
+    # coordinates = models.Polygon()
+
+
+class Person(models.Model):
+    """
+    Person model
+    Fields:
+        `first_name`: first name of a person
+
+        `last_name`: last name of a person
+
+        `ethnicity`: ethnicity of a person, optional field
+
+        `date_of_birth`: date of birth
```

Comment:
You will want to add a hint for what format you are expecting. You can use `verbose_name` attribute set to something like "Date of birth: YYYY-MM-DD".